hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/Route.java

```diff
@@ -38,10 +40,12 @@ public void configure() {
 
         from("jetty:http://{{hostname}}:{{port}}/{{endpoint}}").routeId("hnsecure-route")
             .log("HNSecure received a request")
+            //.process(validator)
```

Comment:
I assume this is commented out because it's dealt with in the other branch.

Reply:
Correct